src/actinia_core/core/common/keycloak_user.py

```diff
@@ -0,0 +1,323 @@
+# -*- coding: utf-8 -*-
```

Comment:
Please move file one level up, out of `common` folder. This folder was meant for modules / classes which are used by actinia plugins as well, although this wasn't communicated well enough ;)

Reply:
But two modules (module-plugin and parallel-plugin) are using ActiniaUser. So maybe they should also include keycloak and use this new user class.

Reply:
alright then.